test-nokogumbo.rb

```diff
@@ -53,27 +54,27 @@ def test_nil
     def test_macroman_encoding
       mac="<span>\xCA</span>".force_encoding('macroman')
       doc = Nokogiri::HTML5(mac)
-      assert_equal '<span>&#xA0;</span>', doc.at('span').to_xml
+      assert_equal "<span> </span>", doc.at("span").to_xml
```

Comment:
This change makes it difficult to see what character is expected. It's also not related to  testing your actual change.